pa_volume.c

```diff
@@ -311,8 +322,11 @@ static void parse_args(int argc, char **argv)
     client = argv[optind++];
   if(optind < argc) {
     char *endptr = NULL;
+    if(strcmp(argv[optind], "toggle") == 0){
+      toggle_mute = true;
+    }
     volume = strtod(argv[optind], &endptr);
-    if(*endptr != '\0') {
+    if(*endptr != '\0' && !toggle_mute) {
```

Comment:
I'd rather see these lines moved into the else block of the if in line 325 and not add the && !toggle_mute to this line.